statsmodels/stats/anova.py

```diff
@@ -325,9 +347,7 @@ def anova_lm(*args, **kwargs):
         model = args[0]
         return anova_single(model, **kwargs)
 
-    try:
-        assert typ in [1,"I"]
-    except:
+    if typ not in [1,"1"]:
```

Comment:
Did this change upper-case-i to “1” intentionally?  Nitpick: Space after comma pls.

Reply:
Thank you for that catch...I typoed that "1" in there.  Fixed the spacing.